src/pages/dashboard.tsx

```diff
@@ -76,29 +84,29 @@ export function TableHero() {
   }, []);
 
   return (
-    <div className="p-4 md:p-10 rounded-2xl bg-gray-900">
-      <h1 className="text-gray-300 text-5xl font-semibold mb-2 mt-2 px-10">
+    <div className="p-4 md:p-10 rounded-2xl my-12">
+      <h1 className="text-white hover:text-secondary text-4xl md:text-5xl font-bold md:mb-4 capitalize">
         Team Members
       </h1>
       {loading ? (
         <p className="text-gray-300">Loading...</p>
       ) : (
-        <div className="overflow-hidden">
-          <Table className="text-gray-300 text-sm my-4 mx-8 border-separate border-spacing-0">
+        <div className="overflow-auto">
+          <Table className="text-lg border-separate border-spacing-0 rounded">
             <TableHead>
-              <TableRow className="bg-gray-700">
+              <TableRow className="bg-black">
                 {tableHeaders}
               </TableRow>
             </TableHead>
             <TableBody>
               {users.map((user) => (
                 <TableRow
                   key={user.slack_id}
-                  className="text-gray-300 hover:bg-gray-200 hover:bg-opacity-50 transition-colors duration-200 border-b border-gray-300"
+                  className="text-black border-b border-gray-300"
                 >
                   {
                     userColumns.map((keyName, i) =>
-                      <TableCell key={keyName + i} className="bg-gray-600 px-4 py-2 text-sm">
+                      <TableCell key={keyName + i} className="bg-white px-4 py-2 text-base text-left">
```

Comment:
I think there's a class for this, somethign like bg-dark-bg and bg-light-bg, check the tailwind config file